### PR TITLE
Test fixes

### PR DESCRIPTION
**Pull request by Adirelle**
Some minor fixes so the unit tests can run on windows and/or with missing PHP extensions.

### CODE REVIEW

File: Tests/PHPCI/Helper/CommandExecutorTest.php [hunk 1]
```diff
@@ -14,6 +14,10 @@ class CommandExecutorTest extends ProphecyTestCase
 
     protected function setUp()
     {
+        if(IS_WIN) {
```
Comment (mavimo) [inline Tests/PHPCI/Helper/CommandExecutorTest.php]: space after if before (

**Comment by tvbeek** [comment]
@dancryer can you give the result of the build so @Adirelle can fix it?

**Comment by mavimo** [comment]
PHP Notice:  Undefined variable: strings in /PHPCI/Helper/Lang.php on line 161

## PHPUnit failed tests

Error: PHPCI\Plugin\Tests\Command\CreateAdminCommandTest::testExecute
Error: PHPCI\Plugin\Tests\Command\InstallCommandTest::testAutomaticInstallation
Error: PHPCI\Plugin\Tests\Command\InstallCommandTest::testDatabaseHostnameConfig
Error: PHPCI\Plugin\Tests\Command\InstallCommandTest::testDatabaseNameConfig
Error: PHPCI\Plugin\Tests\Command\InstallCommandTest::testDatabaseUserameConfig
Error: PHPCI\Plugin\Tests\Command\InstallCommandTest::testDatabasePasswordConfig
Error: PHPCI\Plugin\Tests\Command\InstallCommandTest::testPhpciUrlConfig
Error: PHPCI\Plugin\Tests\Command\InstallCommandTest::testAdminEmailConfig
Error: PHPCI\Plugin\Tests\Command\InstallCommandTest::testAdminUserameConfig
Error: PHPCI\Plugin\Tests\Command\InstallCommandTest::testAdminPasswordConfig
Error: PHPCI\Plugin\Tests\Helper\CommandExecutorTest::testFindBinary_ReturnsPathInSpecifiedRoot
Error: PHPCI\Plugin\Tests\Util\ExecutorTest::testExecutePlugin_LogsFailureForNonExistentClasses
Error: PHPCI\Plugin\Tests\Util\ExecutorTest::testExecutePlugin_LogsFailureWhenExceptionsAreThrownByPlugin
Error: PHPCI\Plugin\Tests\Util\ExecutorTest::testExecutePlugins_CallsEachPluginForStage

I hope this can help

**Comment by Adirelle** [comment]
@mavimo: cannot reproduce this bug. Moreover, I don't understand how it can happen unless the lang.<language>.php exists but does not define the $strings global. With which version of PHP is this failing ?

**Comment by mavimo** [comment]
@Adirelle this happen on PHP CI server, I have no idea what's the PHP version, try to rebase/merge master on it and push...

**Comment by Adirelle** [comment]
It is already based on latest master.

@dancryer Any clue ?

**Comment by mavimo** [comment]
@dancryer can you manually trigger a build on the last commit here?

**Pull request closed without merging**